src/Microsoft.Azure.SignalR.Serverless.Protocols/ServerlessMessage.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.SignalR.Serverless.Protocols
+{
+    public abstract class ServerlessMessage
+    {
+        [JsonProperty(PropertyName = "type")]
+        public int Type { get; set; }
+    }
+
+    public class InvocationMessage : ServerlessMessage
```

Comment:
>InvocationMessage [](start = 17, length = 17)

It seems in Runtime, `InvocationMessage` doesn't have type property, Shall we make RT and here consistent?

Reply:
`InvocationMessage` in RT is not the `InvocationMessage` here. That in RT is just a wrapper over the real `InvocationMessage`